test/extended/util/test.go

```diff
@@ -453,6 +453,21 @@ var (
 // checkSyntheticInput selects tests based on synthetic skips or focuses
 func checkSyntheticInput() {
 	checkSuiteSkips()
+	checkSkipTestsEnvVar()
+}
+
+func checkSkipTestsEnvVar() {
+	envvar, exists := os.LookupEnv("SKIP_TESTS")
+
+	if exists {
+		skipTests := strings.Split(envvar, " ")
```

Comment:
Splitting on space means we can only skip based on single-word matches. While that works for our case it might be better to have this be more generic... Maybe split on `"\n"`?

Reply:
aye, just used space as a WIP to see what you thought about it in general